Lib/imghdr.py

```diff
@@ -42,6 +45,7 @@ def test_jpeg(h, f):
 tests.append(test_jpeg)
 
 def test_png(h, f):
+    """PNG data"""
```

Comment:
Here and the next docstrings, please end them with a dot. But also, express what the function does.

E.g. "Verify if the image is a PNG."